views/dashboard.html

```diff
@@ -1,34 +1,31 @@
 {% extends "layout.html" %}
 
 {% block body %}
-<div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
-  <h2 class="sub-header">The Latest 100</h2>
-  <div class="table-responsive">
-    <table class="table table-striped">
-      <thead>
+<div class="col-sm-3 col-sm-offset-4 col-md-3 main">
+
+  <p class="h3" align="center">Executive Summary</p>
+    <div class="table-responsive center-block" >
+      <table class="table">
         <tr>
-          <th>Website</th>
-          <th>Ad Network</th>
-          <th>Status Code</th>
-          <th>File Type</th>
-          <th>Complete Time</th>
-          <th>File Size</th>
+          <td align="center" style="border:none">
+            <p style="font-weight: bold; font-size: 2.5em">{{assets}}</p>
+            <p class="">Assets Benchmarked</p>
+          </td>
         </tr>
-      </thead>
-      <tbody>
-        {% for record in records %}
+      </table>
+      <table class="table">
```

Comment:
Rather than a table, can you use similar styling to what is in the add-on?

Markup: https://github.com/FrancescoSTL/Site-Sonar/blob/master/views/userReport.html#L29
Styling: https://github.com/FrancescoSTL/Site-Sonar/blob/master/styles/userReport.css#L56  etc, etc